WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -823,9 +829,6 @@ extension OrderListViewController: UITableViewDelegate {
         }
 
         syncingCoordinator.ensureNextPageIsSynchronized(lastVisibleIndex: itemIndex)
-        if indexPath == selectedIndexPath {
-            highlightSelectedRowIfNeeded()
-        }
```

Comment:
this move makes sense to me. Just to make sure there are no other reasons for the highlight logic here, double checking with @joshheald.

Reply:
> Did anyone check the deeplink highlighting behaviour?

I did test those with both on-screen and out-of-screen order deeplinks and the rows are highlighted correctly (out-of-screen still doesn't scroll automatically as https://github.com/woocommerce/woocommerce-ios/issues/11789)